web/html/src/manager/virtualization/nets/network-properties.js

```diff
@@ -46,8 +47,68 @@ type Props = {
 
 function clearFields(initialModel, setModel) {
   if (initialModel) {
-    // Flatten the model for the form
-    let flattened = flattenModel(initialModel);
+    let definition = initialModel;
+
+    // Convert some int properties to string since validators only work with strings
```

Comment:
Instead of this we should fix the validators in my opinion. Otherwise we'll have this workaround all over the place pretty soon.

Reply:
Created a card for this: https://github.com/uyuni-project/uyuni/issues/3391